content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,485 @@
+
+```jsx
+// use 'sharetribe-flex-integration-sdk' here to create and integration SDK instance
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+  clientSecret: '<Client secret>',
+  tokenStore: sharetribeSdk.tokenStore.memoryStore(),
+});
+```
+
+## Log in using valid credentials
+
+```jsx
+sdk.login({ username: '<User email>', password: '<User password>' });
+```
+
+## Exchange the access token for a trusted token
+
+```jsx
+sdk.exchangeToken().then((res) => {
+  var store = sharetribeSdk.tokenStore.memoryStore();
+  store.setToken(res.data);
+});
+```
+
+## Create an instance of the SDK using the trusted token
+
+```jsx
+var sdkTrusted = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+  tokenStore: sharetribeSdk.tokenStore.memoryStore(),
+});
+```
+
+</Steps>
+
+Now you can use `sdkTrusted` to make requests that rely on a secure
+context, e.g. deleting a user with
+[Marketplace API](https://www.sharetribe.com/api-reference/marketplace.html#delete-user).
+
+<Callout type="info">
+  Requests that require a trusted SDK rely on valid login credentials
+  appropriate for the request. For example, to delete a user you must
+  log in with that user's credentials.
+</Callout>
+
+## Example usage
+
+Here are a few examples demonstrating common uses and capabilities of
+the SDKs.
+
+### Logging in
+
+**API**: Marketplace API
+
+Some endpoints require that a user is logged in first, e.g. creating
+listings, viewing their own listings, or initiating inquiry
+transactions.
+
+```jsx
+const sharetribeSdk = require('sharetribe-flex-sdk');
+
+var sdk = sharetribeSdk.createInstance({
+  clientId: '<Client ID>',
+});
+
+sdk
+  .login({ username: '<User email>', password: '<User password>' })
+  .then((_) => {
+    // Request dependent on an authenticated user access token
+  });
+```
+
+### Initiating a transaction
+
+**API**: Marketplace API
+
+The process of initiating a transaction is highly variable because it
+depends entirely on transaction flow customizations and the required
+paramters of the transition.
+
+For this example, we'll use the default-booking process, and choose
+`transition/request-payment` as the first step. This requires that:
+
+- The customer is logged in
+- A token exchange is made to get a trusted user access token
+- The request includes the following parameters
+  - `transition`
+  - `processAlias`
+  - `params`
+    - `listingID`
+    - `bookingStart`
+    - `bookingEnd`
+    - `lineItems`
+
+<Callout type="info">
+  This example uses the **minimum** required parameters per the
+  transition defined in the transaction process. You'll likely want to
+  include more information during this step, e.g. updating the
+  transaction's extended data. See the [Marketplace API reference
+  documentation](https://www.sharetribe.com/api-reference/marketplace.html#initiate-transaction)
+  for more information.
+</Callout>
+
+```jsx
```

Comment:
Could add SDK import to the beginning here to be consistent with the other examples, i.e.
`const sharetribeSdk = require('sharetribe-flex-sdk');`